control.tf

```diff
@@ -2,13 +2,16 @@ locals {
   control_team = [
     "bmagyar",
     "christophfroehlich",
+    "destogl",
+    "saikishor",
   ]
   control_repositories = [
     "control_msgs-release",
     "control_toolbox-release",
     "gazebo_ros2_control-release",
     "ign_ros2_control-release",
     "kinematics_interface-release",
+    "mujoco_ros2_simulation-release",
```

Comment:
New release repositories need a merged `source` stanza in rosdistro first. Let's drop that part of this PR and get the new members in while the `mujoco_ros2_simulation` package is reviewed for inclusion in ROS.


```suggestion
```